docs/contributing/compose.rst

```diff
@@ -0,0 +1,75 @@
+.. Copyright 2017 tsuru authors. All rights reserved.
+   Use of this source code is governed by a BSD-style
+   license that can be found in the LICENSE file.
+
++++++++++++++++++++++++++++++++++++++++++++++++
+Building a development environment with Docker Compose.
++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
I think the `+++++++++++++++++++++++++++++++++++++++++++++++` lines need to be the same length as `Building a development environment with Docker Compose.` else sphinx complains